lib/pagseguro/transaction/serializer.rb

```diff
@@ -32,13 +32,13 @@ def serialize_status_history
 
       private
       def serialize_general(data)
-        data[:code] = xml.css("transaction > code").text
-        data[:reference] = xml.css("transaction > reference").text
-        data[:type_id] = xml.css("transaction > type").text
-        data[:payment_link] = xml.css("transaction > paymentLink").text
-        data[:status] = xml.css("transaction > status").text
+        data[:code] = xml.css("code")[0].text
```

Comment:
This way (`xml.css("code").text`) does not work, 6 specs broken.

If you can change the specs and make it passes let me know.